class DataNotFoundError(Exception):
    pass

class DataExistsError(Exception):
    pass
